enforcer-rules/src/it/return-type-mismatch/invoker.properties

```diff
@@ -1,2 +1,3 @@
 invoker.goals=verify
 invoker.buildResult=failure
+invoker.java.version=1.8
```

Comment:
The return-type-mismatch test case is for the protobuf-java problem around `java.nio.Buffer`. The problem does not occur in Java 11.